go/cmd/dolt/commands/sql.go

```diff
@@ -436,7 +478,8 @@ func runShell(ctx context.Context, se *sqlEngine, dEnv *env.DoltEnv) error {
 		}
 
 		if sqlSch, rowIter, err := processQuery(ctx, query, se); err != nil {
-			shell.Println(color.RedString(err.Error()))
+			verr := formatQueryError(query, err)
+			shell.Println(verr.Verbose())
```

Comment:
You forgot the red!

Reply:
verr.Verbose() already prints the first line as red.